src/main/java/com/atlassian/bitbucket/jenkins/internal/config/BitbucketServerConfiguration.java

```diff
@@ -61,10 +61,14 @@ public BitbucketServerConfiguration(
             String baseUrl,
             @Nullable String credentialsId,
             @Nullable String id) {
-        this.adminCredentialsId = requireNonNull(adminCredentialsId);
+        this.adminCredentialsId = requireNonNull(adminCredentialsId, "adminCredentialsID");
```

Comment:
I think we should be consistent, so either add for all of the fields or for none of them